app/views/content/train-to-be-a-teacher/initial-teacher-training.md

```diff
@@ -28,6 +28,8 @@ You’ll also have a mentor to support you in your school placements and learnin
 
 There are full-time and part-time teacher training courses available.
 
+As part of selecting your teacher training course, you will need to decide if you want to train to teach at a primary or secondary level. [Learn about deciding who to teach](/train-to-be-a-teacher/who-do-you-want-to-teach).
```

Comment:
@philspray @EllieNodder 

Not sure whether this makes sense on this page - this info is about the course itself, not choosing it?

Reply:
@philspray @gemmadallmandfe might it make more sense to put something about primary/secondary in the classroom placements section? Something along the lines of 'where you complete your classroom placements will depend on whether you've chosen a primary or secondary teacher training course'?